introduction.md

```diff
@@ -0,0 +1 @@
+My name is Jesse.  I'm about kickin' ass, I'm about takin' names, and we're about makin' games.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.